documentation-site/components/yard/README.md

```diff
@@ -0,0 +1,254 @@
+# react-view
+
+**React View is an interactive playground, documentation and code generator** for your  React components. All that packaged in a single component that can be dropped into any application.
+
+Use it to document your component APIs. Users can then live-edit component props and grab the generated ready-to-use source code.
+
+[**Check this demo.**](https://baseweb.design/components/input/)
+
+All you need to do is describing your component with a configuration file and React View will do the rest.
+
+React View has **three main parts**:
+
+- **rendered component**
+- **knobs** - each prop is mapped into an input, radio group, textarea or checkbox, those values are live-editable
+- **code** - gets updated as you edit the knobs, it can be also edited directly
+
+## Always Synchronized
+
+These three parts are **3-way synchronized**. 😱 What does it mean?
+
+- An interaction with the rendered component updates the knobs and code.
+- Editing the knobs updates the rendered component and code.
+- Editing the code updates the knobs and rendered component.
+
+## Installation
+
+    yarn add react-view
+
+## Building Blocks
+
+- **react-view basic**
+    - Comes with no UIs or styling, focused on the data.
+    - Customization of parsing/code generation for all PropTypes.
+    - Adding custom PropTypes.
+    - Adding an optional Provider (Theme) parsing / code generation.
+    - Render prop to build your own UIs.
+    - Exports basic UIs for Compiler, Editor, Error.
+- **react-view/minimalistic**
+    - Comes with a basic minimal styling but should be useful out of the box.
+    - Allows adding an optional Provider (Theme) parsing / code generation.
+- **react-view/base**
+    - Comes with Base Web, polished, a drop-in replacement for yard.
+    - By default assumes / supports Base Web provider (theme) structure.
+    - Requires installing / setting up baseui and styletron.
+    - Fully supports `PropTypes.Overrides`.
+
+## react-view basic
+
+```jsx
+import {
+  View,
+  Compiler,
+  Editor,
+  Error,
+  PropTypes,
+} from 'react-view';
+
+<View
+  componentName="Button"
+  componentPlaceholder={<div />}
```

Comment:
what is this for? render a spinner while initializing?

```suggestion
  componentPlaceholder={<div>Loading...</div>}
```

Reply:
Yea!